cli/tracker.php

```diff
@@ -116,7 +116,8 @@
 
 try
 {
-	// Set the logger for the application.  We're doing it here because there is a recursion issue with correct service resolution that needs to be fixed.
```

Comment:
All you really needed to do was take one space out from between the two sentences.  No need for a multi-line comment.  Or to make a little more room start the second sentence as "We do it here because..."